doc/distrib/NodeHelpFiles/_review/Revit.Elements.Views.View.IsCategoryHidden.md

```diff
@@ -0,0 +1,8 @@
+## In Depth
+`View.IsCategoryHidden` returns a true or false value to indicate if the category has been hidden in the given view. Note, if a view has a view template assigned, the view template must be used.
+
+In the example below, the active view's view template from the current document is retrieved. `View.IsCategoryHidden` reports if the wall category is hidden.
```

Comment:
For readability, suggest rephrasing this as "In the example below, the view template of the active view from the current document is retrieved...." Sorry for being picky! Active view's view template is a hard to read.